site/content/en/docs/formats/common_semantic_segmentation.md

```diff
@@ -0,0 +1,83 @@
+---
+title: 'Common Semantic Segmentation'
+linkTitle: 'Common Semantic Segmentation'
+description: ''
+weight: 2
+---
+
+## Format specification
+
+CSS format specification is available [here](https://github.com/openvinotoolkit/open_model_zoo/blob/master/tools/accuracy_checker/openvino/tools/accuracy_checker/annotation_converters/README.md#supported-converters).
+
+Supported annotation types:
+- `Masks`
+
+## Import Common Semantic Segmentation dataset
+
+A Datumaro project with a CSS source can be created in the following way:
+
+``` bash
+datum create
+datum import --format common_semantic_segmentation <path/to/dataset>
+```
+
+Extra options for adding a source in the COCO format:
```

Comment:
```suggestion
Extra import options:
```

COCO doesn't seem to be related to this format, because it stores masks in a different encoding. Also, they won't be located by this format, if the dataset follows the default Datumaro COCO layout.